lib/diagnostic.js

```diff
@@ -18,16 +18,17 @@
 // on line 3?
 //
 //     `true` or `false`
+true
 //
-let access = null; // replace null with your response
+let access = deathstar.laserQuantity; // replace null with your response
 //
 
 // Which scope(s) may access the variable `laserQuantity`? Choose one or many
 // of the following:
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-let scopes = [/* your response here */];
+let scopes = ['deathstar'   ];
```

Comment:
Close. The `fire` function has access to laserQuantity, because it is defined in the same scope as `laserQuantity`. 
.5/1